src/S3.cpp

```diff
@@ -1,11 +1,12 @@
-#include "S3.h"
+#include <S3.h>
 
 //#define DEBUG
 
 using namespace Aws::S3;
 
-Aws::SDKOptions options;
 namespace cirrus {
+Aws::SDKOptions options;
+
 static bool called = false;
 void s3_initialize_aws() {
```

Comment:
Looking at this function again, I think instead of throwing an std::runtime_error that S3 is already active, maybe just return without doing anything.

Reply:
I prefer to force the user to call this only when needed.